object/search.go

```diff
@@ -263,6 +278,11 @@ func (f *SearchFilters) addRootFilter() {
 	f.addFlagFilter(fKeyPropRoot)
 }
 
+// AddRootFilter adds filter to object types.
+// With this filter will be returned only REGULAR type objects that are not split or that are the top level root objects in a
+// split hierarchy. This includes objects not present physically, like large objects split into smaller
+// objects without a separate top-level root object. Objects of other types like StorageGroups and
+// Tombstones will not be shown.
```

Comment:
but in fact, i would try to describe it as "objects that have been created by a user explicitly". it is the closest to the original meaning and that is simpler to understand IMO